src/data-loader.ts

```
@@ -59,20 +83,20 @@
 	message: z.object({
 		usage: z.object({
 			input_tokens: z.number(),
 			output_tokens: z.number(),
 			cache_creation_input_tokens: z.number().optional(),
 			cache_read_input_tokens: z.number().optional(),
 		}),
-		model: z.string().optional(), // Model is inside message object
-		id: z.string().optional(), // Message ID for deduplication
+		model: modelNameSchema.optional(), // Model is inside message object
+		id: messageIdSchema.optional(), // Message ID for deduplication
 	}),
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)